config/routes.rb

```diff
@@ -120,13 +120,16 @@
   get 'subregion_options' => 'collection_drafts#subregion_options'
 
   scope module: :proposal do
+    get '/collection_draft_proposals/queued_index' => 'collection_draft_proposals#queued_index', as: 'queued_index_collection_draft_proposals'
```

Comment:
Adding the routes here means they get matched correctly.  Defining them after means they match show/:id and generate errors.